### ISSUE TITLE
Should high-res F-compset grid combos use tx12 mask?

### ISSUE THREAD

**Issue by zarzycki** [ESMCI/ccs_config_cesm#52]
Getting my feet wet with adding some new grids with nuopc. With a 15km MPAS grid + gx1v7 mask (a homebrew, admittedly, but templated using existing MPAS grids in CIME) I get the below 10m wind field for Hurricane Harvey that clearly shows a low resolution lnd/ocn mask.

```
  <model_grid alias="mpasa15natl_mpasa15natl" not_compset="_POP">
    <grid name="atm">mpasa15natl</grid>
    <grid name="lnd">mpasa15natl</grid>
    <grid name="ocnice">mpasa15natl</grid>
    <mask>gx1v7</mask>
  </model_grid>
```

![gx1v7](https://user-images.githubusercontent.com/10341827/180066389-937a168e-b815-4797-8290-53b7cd770dfe.png)

You can see the "blockiness" associated with the land/sea mask and the corresponding fluxes provided to CAM (in this case, the land tiles are draggier than the ocean tiles, making for a sharp gradient in surface wind).

If I change the mask to tx0.1v2 in modelgrid_aliases_nuopc.xml...

```
  <model_grid alias="mpasa15natl_mpasa15natl" not_compset="_POP">
    <grid name="atm">mpasa15natl</grid>
    <grid name="lnd">mpasa15natl</grid>
    <grid name="ocnice">mpasa15natl</grid>
    <mask>tx0.1v2</mask>
  </model_grid>
```

![tx01v2](https://user-images.githubusercontent.com/10341827/180066411-1b1d20a3-5b82-4014-bb6d-120dc98a0a19.png)

I get a much better lnd/ocn mask more amenable to the high-resolution grid.

Perhaps a dumb question (feel free to close it if is!), but for all `not_compset="_POP"` combos that are of higher resolution than gx1v7, should we be using tx0.1v2 by default?

For example, all the MPAS grid configs (even the ones used for Earthworks, e.g., mpasa3p75_mpasa3p75_mg17?) are using the gx1v7 for masking by default, which would seem to have a similar issue as above.

**Comment by adamrher** [comment]
> Perhaps a dumb question (feel free to close it if is!), but for all not_compset="_POP" combos that are of higher resolution than gx1v7, should we be using tx0.1v2 by default?

yes, hi-res F-cases should use the tx0.1v2 or tx0.1v3 ocean mask. Although I see many instances of using the gx1v7 mask w/ a hi-res atmos grid ... I presume it's just habit, but we should break this habit.
